packages/react-components/src/components/RichTextEditor/RichTextEditor.test.tsx

```diff
@@ -65,15 +65,16 @@ describe('RichTextEditor should be shown correctly', () => {
         styles={{ minHeight: '1rem', maxHeight: '1rem' }}
       />
     );
-    const boldButton = screen.queryByLabelText('Bold');
-    const italicButton = screen.queryByLabelText('Italic');
-    const underlineButton = screen.queryByLabelText('Underline');
-    const bulletListButton = screen.queryByLabelText('Bulleted list');
-    const numberListButton = screen.queryByLabelText('Numbered list');
-    const indentDecreaseButton = screen.queryByLabelText('Decrease indent');
-    const indentIncreaseButton = screen.queryByLabelText('Increase indent');
+    // const boldButton = screen.queryByLabelText('Bold');
```

Comment:
Do we still need this commented boldButton?

Reply:
Removed!